tests/conftest.py

```
@@ -1,24 +1,201 @@
+
+            if heap_info:
+                _LIFECYCLE_METRICS.append(
+                    {
+                        "test": request.node.name,
+                        "used_heap_mb": heap_info["used"],
+                        "total_heap_mb": heap_info["total"],
+                        "limit_mb": heap_info["limit"],
+                    }
+                )
+        except Exception:
+            pass
```

Ruff (0.16.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)